eq-author/package.json

```diff
@@ -93,19 +93,15 @@
     "draft-js-plugins-editor": "latest",
     "draft-js-raw-content-state": "latest",
     "draftjs-filters": "^1.0.0",
-    "enzyme-to-json": "latest",
     "firebase": "latest",
     "firebaseui": "latest",
     "graphql": "latest",
     "graphql-anywhere": "latest",
-    "graphql-iso-date": "latest",
```

Comment:
Don't we still use this? https://github.com/ONSdigital/eq-author-app/blob/master/eq-author-api/schema/resolvers/base.js#L1

Reply:
This is for eq-author, eq-author-api `package.json` still has `graphql-iso-date` as dependency.

Reply:
This is for eq-author, eq-author-api `package.json` still has `graphql-iso-date` as dependency.